src/components/tabs/tabs.ts

```diff
@@ -77,11 +120,25 @@ export class MdTabGroup {
   /** When the focus index is set, we must manually send focus to the correct label */
   set focusIndex(value: number) {
     this._focusIndex = value;
+
+    this._onFocusChange.emit(this._getChangeEvent(value));
+
     if (this._labelWrappers && this._labelWrappers.length) {
       this._labelWrappers.toArray()[value].focus();
     }
   }
 
+  private _getChangeEvent(index: number): MdTabChangeEvent {
+    if (this._isInitialized) {
```

Comment:
This `isInitialized` check should be used for emitting the event, otherwise you're just going to emit an event with a `null` argument.